modules/ROOT/pages/_partials/data-sync/server-realtime-updates-amq.adoc

```diff
@@ -42,26 +48,32 @@ spec:
   plan: brokered-single-broker
 ----
 
-Create the `AddressSpace`.
-
+. Create the `AddressSpace`.
++
 ----
 oc create -f brokered-address-space.yaml
 ----
 
-Get the details of the `AddressSpace`.
-
+. Check the status of the address space:
++
 ----
 oc get <`AddressSpace` name> -o yaml
 ----
++
+The output displays information about the address space, including details required for connecting applications.
 
-The output will display information including various connection details for connecting applications.
+.Additional Information
 
-See https://access.redhat.com/documentation/en-us/red_hat_amq/7.2/html-single/using_amq_online_on_openshift_container_platform/index#create-address-space-cli-messaging[Creating `AddressSpace`s using the command line] for more information.
+See https://access.redhat.com/documentation/en-us/red_hat_amq/7.3/html-single/using_amq_online_on_openshift_container_platform/index#create-address-space-cli-messaging[Creating address spaces using the command line] for more information.
 
 == Creating an Address
 
-An `Address` is part of an `AddressSpace` and represents a destination for sending and receiving messages. An `Address` is created and used to represent a topic in MQTT terms. For MQTT, a `topic` `Address` type must be used.
+An adress is part of an `AddressSpace` and represents a destination for sending and receiving messages. 
+You use an `Address` to represent an MQTT topic. 
+For MQTT, you must use a `topic` adress type.
```

Comment:
I think there is some redundancy in L72 and L73 and it might be a little confusing.

I think we could condense those down into one line like 
```
Use a `topic` type `Address` to represent an MQTT topic
```

Reply:
alternatively 

```
Use an `Address` with type `topic` to represent an MQTT topic.
```